DutchFilter/sections/specific.txt

```diff
@@ -81,6 +81,7 @@ vlinderscrime.nl##a[href="http://mariska-o.nl/"]
 vlinderscrime.nl##a[href="http://www.fashioncarmen.nl"]
 klusidee.nl##td > .hide-mobile[style*="728px; height:"]
 klusidee.nl##td.td-ad
+klusidee.nl###wa_cmpOverlay
```

Comment:
Please move to Annoyances/cookies. When you visit the site without AdGuard, you will see cookie message and when the message is accepted, the overlay is gone.

Reply:
Thanks for the explanation. Yesterday I was thinking where it should be.